### PR TITLE
Add post mortem post for todays failed deployment

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: _posts/deployments/2022-12-01-post-mortem.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+category: deployments
+---
+
+After yesterday’s deployment, we faced a downtime in our [reference server](https://build.opensuse.org).
+We want to share with you a detailed explanation of what happened.
+
+## Impact
+
+Our [reference server](https://build.opensuse.org) was offline from 09:50 (CEST) during 13 minutes.
+Our instance responded to every request with a downtime maintenance message.
```
[the remark this answers is not in the document]
Comment (danidoni) [reply _posts/deployments/2022-12-01-post-mortem.md:13]: I would remove anything between `with a` and `maintenance message` so it reads `Our instance responded to every request with a maintenance message.`

File: _posts/deployments/2022-12-01-post-mortem.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+No one was able to work with the API or user interface during that time.
+
+
+## Root Causes
+
+Around **09:49** (CEST), we ran a deployment.
+The script used to run the deployment alerted that the deployment finished with an error.
+From this time on, our instance responded to every request with a downtime maintenance message.
```
[the remark this answers is not in the document]
Comment (danidoni) [reply _posts/deployments/2022-12-01-post-mortem.md:21]: I would remove anything between `with a` and `maintenance message` so it reads `Our instance responded to every request with a maintenance message.`

File: _posts/deployments/2022-12-01-post-mortem.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+## Resolution
+
+Updating the ruby packages.
+
+
+## Action Items
+
+- The script used to deploy will be improved to alert when other packages need to be updated, not only the `obs-api` package.
```
Comment (dmarcoux) [inline _posts/deployments/2022-12-01-post-mortem.md:45]: We should track this work in the [ansible-obs](https://github.com/openSUSE/ansible-obs) repository.

Comment (eduardoj) [reply _posts/deployments/2022-12-01-post-mortem.md:45]: I created this issue in the ansible-obs repository: https://github.com/openSUSE/ansible-obs/issues/66

File: _posts/deployments/2022-12-01-post-mortem.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/deployments/2022-12-01-post-mortem.md]: I mixed both suggestions into: `From that time on, our instance responded to every request with a maintenance message.`. Does it work for both of you?

File: _posts/deployments/2022-12-01-post-mortem.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/deployments/2022-12-01-post-mortem.md]: I opted for @danidoni 's suggestion. Does it work for you, @hellcp-work ?

**Comment by hennevogel** [comment]
I have some more changes...

**Review by hennevogel — APPROVED** [review]

**Pull request merged**